source/base/data_out_base.cc

```diff
@@ -5860,10 +5897,22 @@ namespace DataOutBase
                  unsigned int,
                  std::string,
                  DataComponentInterpretation::DataComponentInterpretation>>
-      &nonscalar_data_ranges)
+      &             nonscalar_data_ranges,
+    const VtkFlags &flags)
   {
     AssertThrow(out, ExcIO());
 
+    // If the user provided physical units, make sure that they don't contain
+    // quote characters as this would make the VTU file invalid XML and
+    // probably lead to all sorts of difficult error messages. Other than that,
+    // trust the user that whatever they provide makes sense somehow.
+    for (const auto &unit : flags.physical_units)
```

Comment:
```
/home/runner/work/dealii/dealii/source/base/data_out_base.cc:5909:22: error: unused variable ‘unit’ [-Werror=unused-variable]
```
and similar in the other places.

Reply:
Blech. I have to `(void)` these variables for release mode apparently...